src/services/bridge/pair-client.ts

```diff
@@ -31,6 +31,12 @@ export const pairClient = async ({
     const response = await fetch(`${bridgeURL}/request/${requestUUID}`);
 
     if (!response.ok) {
+      if (response.status == 404) {
+        return {
+          success: false,
+          error: handleError({ message: "input_error" }),
```

Comment:
```suggestion
          error: handleError({ code: "input_error" }),
```
in general we use the convention `message` for human-readable messages and `code` for error codes. also a good idea to type error codes